src/components/Launch/LaunchWorkflowForm/DatetimeInput.tsx

```diff
@@ -8,34 +9,33 @@ import { Moment, utc as moment } from 'moment';
 import * as React from 'react';
 import { InputProps } from './types';
 
-function defaultDate() {
-    return moment()
-        .startOf('day')
-        .toISOString();
-}
-
 /** A form field for selecting a date/time from a picker or entering it via
  * keyboard.
  */
 export const DatetimeInput: React.FC<InputProps> = props => {
     const { label, helperText, onChange, value: propValue } = props;
-    const value = typeof propValue === 'string' ? propValue : defaultDate();
+    const value =
+        typeof propValue === 'string' && propValue.length > 0
```

Comment:
Are you restricted from checking the type of this where `InputProps` is defined?

Reply:
`InputProps.value` can be one of several types, so this is guarding against what happens if we pass a type we don't expect.
I have a planned cleanup pass in the future where I may just make all input values strings to simplify the logic a bit (It's also messy in the conversion code that runs on submit)